docs/modules/administration/pages/helm-migration.adoc

```diff
@@ -0,0 +1,31 @@
+= Migrating from Tailor to Helm
+
+Tailor has been developed for OpenShift 3.11. Back in the days, Helm 2 required the use of a privileged Tiller service and did not work well with OpenShift-specific resources. With Helm 3 and OpenShift 4, this situation as changed.
+
+While Tailor also works in an OpenShift 4 cluster, OpenShift has integrated Helm into its product, and Helm has a huge and growing community. Therefore, it is recommended to use Helm instead of Tailor in an OpenShift 4 cluster.
+
+This document will describe how to migrate from Tailor to Helm for OpenDevStack installation.
+
+Tailor is based on OpenShift templates, which define the Kubernetes resources to apply. Helm uses a different templating language / engine, but in the end the templates also describe Kubernetes resources. Therefore, migration effort is relatively low as one only needs to change the syntax of the definition, not the definition itself. Further, there are differences between the CLI of the two tools and not all features of Tailor are available in Helm and vice-versa. Once migration to Helm is complete, it is also recommended to look at the best practices in the Helm community and adopt these.
+
+== Templates preparation
```

Comment:
I was a bit confused by this section initially. At first I thought that `tailor2helm.sh` would convert the OpenShift templates used by Tailor into the chart template files used by Helm. Now I understand this section does not actually provide guidance how to do that conversion, but rather how to deal with the resources already present in the K8s namespace.

Reply:
Exactly, when I was preparing to update from a SonarQube installation done with Tailor to a new version using Helm it was not possible to do this except if the old resurces are annotated and labeled for Helm using the script

Reply:
I modified the page a bit to avoid this confusion